Added .gitignore to omit .DS_Store files

AminuIsrael opened the pull request:
<!-- Love osca? Please consider supporting our collective:
👉  https://opencollective.com/osca/donate -->

#### What does this PR do?

#### Descripe the Task you're trying to complete

#### Add link to your design

#### What are the relevant issues this PR belongs to

#<ISSUE NUMBER> 

#### Any background context you want to add?

#### Add screenshots (Optional)

perriefidelis commented:
Hi, @AminuIsrael thanks for sending in a PR, but can you properly describe what exactly you're trying to fix?

AminuIsrael commented:
Hi. @perriefidelis, I noticed the repository had an unwanted file **.DS_store** I deleted them in each subdirectory and added a .gitignore which doesn't push them to the remote repository.

BolajiAyodeji reviewed (approved):
Thanks for this PR!
